### DISCUSSION TITLE
v0.27.4 Patch 1

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/UnblockNeteaseMusic#1306] [Announcements]
## TL; DR;
- Fixed kuwo provider
- Fixed pyncmd provider
- Added `source` element in AudioData

## Fixes
- [feat(hook): unlock music items in search page](https://github.com/UnblockNeteaseMusic/server/commit/7ec83985da8be9a2242a9ca4e24f42dda32dd3cb)
- [feat(provider/pyncmd): migrate to new API](https://github.com/UnblockNeteaseMusic/server/commit/b93969ae79a431eb8d354e442a11d26909d71914)
- [fix(provider/kuwo): add signing param](https://github.com/UnblockNeteaseMusic/server/commit/3609cc43f4d6bedc07bd72d568b9a89c977f85e7)

## Features
- [feat(provider): Add source in AudioData](https://github.com/UnblockNeteaseMusic/server/commit/17cbf970cc3d7b2a0767c0bf9b554dbd6c4f42be)

## Chore
- Update dependencies to the latest

**Full Changelog**: https://github.com/UnblockNeteaseMusic/server/compare/v0.27.4...v0.27.4-patch.1

<hr /><em>This discussion was created from the release <a href='https://github.com/UnblockNeteaseMusic/server/releases/tag/v0.27.4-patch.1'>v0.27.4 Patch 1</a>.</em>
